ff2mpv.js

```diff
@@ -1,40 +1,33 @@
-function onError(error) {
-    console.log(`${error}`);
-}
+const webext = typeof browser === "object" ? browser : chrome;
 
-function ff2mpv(url) {
-    browser.tabs.executeScript({
-        code: "video = document.getElementsByTagName('video');video[0].pause();"
-    });
-    browser.runtime.sendNativeMessage("ff2mpv", { url: url }).catch(onError);
-}
-
-async function getOS() {
-  return browser.runtime.getPlatformInfo().then((i) => i.os);
-}
-
-getOS().then((os) => {
-  var title = os == "win" ? "Play in MP&V" : "Play in MPV (&W)";
```

Comment:
Was there any point to this code? I didn't really get it, so I removed it.

Reply:
I have no recollection of why this was added -- my first guess is that someone on Windows wanted a special case for the accelerator key here, since `alt-V` is maybe already occupied by something on Windows.

Reply:
The current key is `Ctrl+Shift+V`, which I don't really like because that's the same shortcut for pasting in many applications.

Something with `Alt`, like `(Ctrl+)Alt+P` or `(Ctrl+)Alt+M` could work. Or maybe we can just leave it blank. If the use really needs it, they can set it themselves.

Reply:
Agreed. IMO fine to remove entirely.